launch/faulkner.launch

```diff
@@ -0,0 +1,49 @@
+<launch>
+  <arg name="model" default="burger"/>
+  <param name="model" value="$(arg model)"/>
+  
+  <arg name="x_pos" default="0.0"/>
+  <arg name="y_pos" default="0.0"/>
+  <arg name="z_pos" default="0.0"/>
+
+  <arg name="des_x" default="4.0"/>
+  <arg name="des_y" default="0.0"/>
+  <arg name="max_lin_vel" default="0.1"/>
```

Comment:
Please add units/description of what these hardcoded numbers are

Reply:
These are non-calibrated or simulated values. probably we know more or we change these once we do integrated testing.